modules/cluster-logging-eventrouter-deploy.adoc

```diff
@@ -11,7 +11,7 @@ The following Template object creates the Service Account, ClusterRole, and Clus
 
 [NOTE]
 ====
-To use the Event Router, you must enable JSON log parsing using the `MERGE_JSON_LOG` feature. For important information `MERGE_JSON_LOG`, see
+Deploying the Event Router automatically enables JSON log parsing using the `MERGE_JSON_LOG` feature. For important information `MERGE_JSON_LOG`, see
```

Comment:
Fluentd will automatically parse JSON logs that come from Pods whose names match `eventrouter-*`, which is the naming convention used in the example configuration.

Reply:
That is, I don't think we should mention `MERGE_JSON_LOG` here.

Reply:
@richm should I just close this PR?